lib/sna/auth/email.ex

```diff
@@ -0,0 +1,40 @@
+defmodule Sna.Auth.Mail do
+
+  @moduledoc """
+
+  Sna.Auth.Mail will store email and token:
+
+  1. when an user comes and want to use email authentication, an email
+     is firstly checked.
+
+  2. if the mail is valid, a token is generated and the email is
+     stored temporarily in the ETS cache. A timer process is started
+     for some minutes.
```

Comment:
A token is already generated in SnaWeb. It's a token that corresponds to a web session, I don't think it needs to be generated in Sna.Auth...

Reply:
Hum. So, how to valid the token if someone valid it from another device? Lot of my mails are on my phone and I use the link from my phone. If I don't have the session shared with my browser, it can't work.